File: src/main/java/me/miki/shindo/management/event/impl/EventReceiveChat.java
package me.miki.shindo.management.event.impl;

import me.miki.shindo.management.event.Event;
import net.minecraft.util.IChatComponent;

public class EventReceiveChat extends Event {

    private IChatComponent message;

    public EventReceiveChat(IChatComponent message) {
        this.message = message;
    }

    public IChatComponent getMessage() {
        return message;
    }

    public void setMessage(IChatComponent newMessage) {
        this.message = newMessage;
    }
}

File: src/main/kotlin/me/miki/shindo/management/event/impl/EventReceiveChat.kt
package me.miki.shindo.management.event.impl

import me.miki.shindo.management.event.Event
import net.minecraft.util.IChatComponent

class EventReceiveChat(
    private var message: IChatComponent,
) : Event() {
    fun getMessage(): IChatComponent = message

    fun setMessage(message: IChatComponent) {
        this.message = message
    }
}
